src/test/ui/feature-gate-isa_attribute.rs

```diff
@@ -0,0 +1,35 @@
+// ignore-aarch64
```

Comment:
@jonas-schievink  @mati865 I don't know if either of you have time to have a look at this. But I can't figure out all the ignores I need to stop this running on architectures other than the arm v4 and v5 targets

Reply:
It should be fine to just run it on one of the Arm architectures

Reply:
Oh that would make things easier, I was trying to test on all relevant arm targets